### PR TITLE
Make `Menu` run without menu state props

### PR DESCRIPTION
**Pull request by namjul**
Should a component break when not all required props are provided? 
Other components like `Dialog` do not break when you don't spread its state-props.

### CODE REVIEW

**Comment by namjul** [comment]
This is also the case with `Radio`.

File: packages/reakit/src/Menu/Menu.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ export const useMenu = createHook<MenuOptions, MenuHTMLProps>({
 
     const { next, previous, orientation } = ancestorMenuBar || {};
     const ancestorIsHorizontal = orientation === "horizontal";
-    const [dir] = options.placement.split("-");
+    const [dir] = (options.placement || '').split("-");
```
Comment (jperasmus) [inline packages/reakit/src/Menu/Menu.tsx]: If you change `''` to `""`, the failing build should pass.

**Comment by diegohaz** [comment]
Thank you @namjul! Could you please add a snapshot test on https://github.com/reakit/reakit/blob/master/packages/reakit/src/Menu/__tests__/Menu-test.tsx to ensure this works without state props?

Also, I'd very much appreciate a PR on Radio as well! :)

**Comment by namjul** [comment]
If i add a test like this:
```
test("render without state props", () => {
  const { baseElement } = render(<Menu />);
  expect(baseElement).toMatchInlineSnapshot(`
    ...
  `);
});
```
typescript errors because of `unstable_hiddenId` is missing in type '{}'.
So how do i write a test for this? Should we change the types?

**Comment by diegohaz** [comment]
Types should reflect at least the minimum required props to make it work, so you can put `// @ts-ignore` above that line in this case.

**Pull request merged**